### PR TITLE
Use 1.19.0-beta.1 version of azure core in request options branch

### PR DESCRIPTION
**Pull request by jianghaolu**

### CODE REVIEW

File: eng/versioning/version_client.txt [hunk 1]
```diff
@@ -58,7 +58,7 @@ com.azure:azure-communication-sms;1.0.3;1.1.0-beta.1
 com.azure:azure-communication-identity;1.1.1;1.2.0-beta.1
 com.azure:azure-communication-phonenumbers;1.0.3;1.1.0-beta.1
 com.azure:azure-containers-containerregistry;1.0.0-beta.3;1.0.0-beta.4
-com.azure:azure-core;1.18.0;1.19.0-beta.2
+com.azure:azure-core;1.19.0-beta.1;1.19.0-beta.2
```
Comment (alzimmermsft) [inline eng/versioning/version_client.txt:61]: Can also use a `beta_com.azure:azure-core;1.19.0-beta.1` tag

**Review by alzimmermsft — APPROVED** [review]

File: eng/versioning/version_client.txt (hunk 1 above)
Comment (jianghaolu) [reply eng/versioning/version_client.txt:61]: Added - I suppose we could do this in the `main` branch too?

**Review by weidongxu-microsoft — APPROVED** [review]

File: eng/versioning/version_client.txt (hunk 1 above)
Comment (weidongxu-microsoft) [reply eng/versioning/version_client.txt:61]: Saw it https://github.com/Azure/azure-sdk-for-java/blob/main/eng/versioning/version_client.txt#L328-L335

Seems added recently.
If we can use that, we can go for main directly for purview account preview release.

**Review by qwordy — APPROVED** [review]

File: eng/versioning/version_client.txt (hunk 1 above)
Comment (jianghaolu) [reply eng/versioning/version_client.txt:61]: Yes - closing this PR then.

**Comment by jianghaolu** [comment]
Closing as `beta_` tag will do it.

**Pull request closed without merging**